nanoFirmwareFlasher.Tool/Program.cs

```diff
@@ -133,19 +137,18 @@ private static void CheckVersion()
 
                 if (latestVersion > currentVersion)
                 {
-                    Console.ForegroundColor = ConsoleColor.DarkYellow;
-                    Console.WriteLine("** There is a new version available, update is recommended **");
-                    Console.WriteLine("** You should consider updating via the 'dotnet tool update -g nanoff' command **");
-                    Console.WriteLine("** If you have it installed on a specific path please check the instructions here: https://git.io/JiU0C **");
-                    Console.ForegroundColor = ConsoleColor.White;
+                    OutputWriter.ForegroundColor = ConsoleColor.DarkYellow;
+                    OutputWriter.WriteLine("** There is a new version available **");
+                    OutputWriter.WriteLine("** Check the update instructions here: https://git.io/JiU0C **");
```

Comment:
Why change the instructions to the developer?
These are meant to simplify one's life and providing the full command options so it's a simple copy paste.
As for the URL here, its specific for when dotnet tools are installed in a non default location. As the original message mentioned. Plus the instruction are spefic to that situation not the general update.

Reply:
I forgot that I'd changed that. If the --suppressversioncheck option is present, the original text is fine. Otherwise it would be a confusing message: you use a frozen version of the tool, and the message is to update the tool. I'll put it back in, with the resolution of the previous item (after PR #377).